english/amnesia/amnesia.md

```diff
@@ -4,5 +4,5 @@ I try some tea, it gets me nowhere.
 It's probably because the marshmellows where filled with horse hooves
 It's definitely throwing off my groove:(
 Then I open my mac and log on to udacity :)
-
```

Comment:
I think the blank line between the text and the options was nice. Could you re-add that? (So add a blank line after your line about horse hooves.)